qa-pipelines/tasks/cf-deploy.sh

```diff
@@ -66,14 +66,19 @@ if [[ ${PROVISIONER} == kubernetes.io/rbd ]]; then
     kubectl get secret -o yaml ceph-secret-admin | sed "s/namespace: default/namespace: ${CF_NAMESPACE}/g" | kubectl create -f -
 fi
 
+if [[ "${EMBEDDED_UAA:-false}" == "true" ]]; then
+    HELM_PARAMS+=(--set "enable.uaa=true")
```

Comment:
you did this in qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh as well
https://github.com/SUSE/cf-ci/pull/270/files#diff-c47082a429cd24ef28f9c5f2a43d896fR191

Reply:
So I did! Fixed in a4822979141bf4d5269f5b1e37ee73f996e67766.